File: app/src/main/java/com/concough/android/models/EntranceStarredQuestionModel.java
package com.concough.android.models;

import java.util.Date;

import io.realm.RealmObject;

/**
 * Created by abolfazl on 7/16/17.
 */

public class EntranceStarredQuestionModel extends RealmObject {
    public EntranceQuestionModel question = null;
    public Date created = new Date();
    public String entranceUniqueId = null;
}

File: app/src/main/java/com/concough/android/models/EntranceStarredQuestionModel.kt
package com.concough.android.models

import io.realm.RealmObject
import java.util.*

/**
 * Created by abolfazl on 7/12/17.
 */
class EntranceStarredQuestionModel: RealmObject() {

    var question: EntranceQuestionModel? = null
    var created: Date = Date()
    var entranceUniqueId: String? = null
}